operate/qa/integration-tests/src/test/java/io/camunda/operate/it/MigrateProcessInstanceOperationZeebeIT.java

```diff
@@ -701,6 +703,100 @@ public void testMigrateProcessInstanceShouldImportMigratedFlowNodes() throws Exc
     assertThat(migratedFlowNode.getBpmnProcessId()).isEqualTo("demoProcess");
   }
 
+  @Test
+  public void testMigrateSubprocessToSubprocess() throws Exception {
```

Comment:
Please don't add anything more to any ITs that utilize the legacy OperateZeebeAbstractIT. These are the main causes of our pipeline long runtime and just have to be manually rewritten to port them to the new templates.